Actor/Info.plist

```diff
@@ -15,7 +15,7 @@
 	<key>CFBundlePackageType</key>
 	<string>FMWK</string>
 	<key>CFBundleShortVersionString</key>
-	<string>1.3</string>
+	<string>1.4</string>
```

Comment:
This is a breaking change. So the version would need be at least 2.0. Unless the changes are put into a differently-named class.